.github/actions/print-logs/action.yml

```diff
@@ -0,0 +1,31 @@
+name: "Print logs"
+description: "Print the log files produced by ci/ci.sh"
+runs:
+  using: "composite"
+  steps:
+    - shell: bash
+      run: |
+        group() {
```

Comment:
The first line of the `run` command is printed as a group header in the Web interface. It could be a meaningful comment, for example: `# Print the log files produced by ci/ci.sh`.

Reply:
Yeah, it's annoying that it's not possible to get rid of this line, but making it contain a comment is a great idea. Fixed.